packages/breadcrumbs/src/elements/Breadcrumb.js

```diff
@@ -18,42 +18,37 @@ import Item from '../views/Item';
  * High-level abstraction for basic Breadcrumb implementations. Accepts all
  * `<ol>` props.
  */
-export default class Breadcrumb extends Component {
-  static propTypes = {
-    children: PropTypes.any
-  };
-
-  renderPage = (page, pageProps, itemProps) => {
-    return <Item {...itemProps}>{itemProps.current ? cloneElement(page, pageProps) : page}</Item>;
-  };
-
-  renderItems = (items, getCurrentPageProps) => {
-    const total = Children.count(items);
-
-    return Children.map(items, (item, index) => {
-      const itemProps = {
-        current: index === total - 1,
-        key: index
-      };
-
-      return hasType(item, Item)
-        ? cloneElement(item, itemProps)
-        : this.renderPage(item, getCurrentPageProps(), itemProps);
-    });
-  };
-
-  render() {
-    const { children, ...breadcrumbProps } = this.props;
-
-    return (
-      <BreadcrumbContainer>
-        {({ getContainerProps, getCurrentPageProps }) => (
-          /* role not needed as `BreadcrumbView` is a navigation landmark. */
-          <BreadcrumbView {...getContainerProps({ role: null })}>
-            <List {...breadcrumbProps}>{this.renderItems(children, getCurrentPageProps)}</List>
-          </BreadcrumbView>
-        )}
-      </BreadcrumbContainer>
-    );
-  }
+const renderPage = (page, pageProps, itemProps) => {
+  return <Item {...itemProps}>{itemProps.current ? cloneElement(page, pageProps) : page}</Item>;
+};
+
+const renderItems = (items, getCurrentPageProps) => {
+  const total = Children.count(items);
+
+  return Children.map(items, (item, index) => {
+    const itemProps = {
+      current: index === total - 1,
+      key: index
+    };
+
+    return hasType(item, Item)
```

Comment:
Is there a way for us to start using context rather than our `hasType` utility? If we're going to move to the hooks we should probably upgrade our usage here as well.

Reply:
yep was thinking that too, I will investigate and see if it's worth bringing in this PR too.

Reply:
So after digging into this I don't think it makes a whole lot of sense to use context. This current code is just checking if the child is already an `<Item />` and just passing it along and if not it wraps the child in an `<Item>`.

Reply:
👍 Sounds good. This is a unique instance since everything is a direct child.